### PR TITLE
Use Material Alert Dialog

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
This PR:
- Remove the custom implementation of AlertDialog
- Use native Material Alert Dialog

|Before|After|
|-|-|
||
|||

### CODE REVIEW

**Review by meenbeese — CHANGES_REQUESTED** [review]
Thanks for helping with this migration. I like the changes but the dialog color in light mode is wrong, can you fix it? 
</br>
Screenshot 2024-04-12 185323

**Comment by Jean-BaptisteC** [comment]
I have done a correction, but we don't have a dialog theme for AMOLED theme

File: app/src/main/java/com/meenbeese/chronos/Chronos.kt [hunk 1]
```diff
@@ -273,6 +273,10 @@ class Chronos : Application(), Player.Listener {
          */
         get() = currentRingtone != null && currentRingtone!!.isPlaying
 
+    fun isDarkTheme():Boolean {
```
Comment (meenbeese) [inline app/src/main/java/com/meenbeese/chronos/Chronos.kt:276]:
```suggestion
    fun isDarkTheme(): Boolean {
```

**Comment by meenbeese** [comment]
> I have done a correction, but we don't have a dialog theme for AMOLED theme

Thanks, that is not a huge concern. I need to rework the entire theme handling in the future anyway.

**Pull request merged**